libs/libmysqlxx/src/Connection.cpp

```diff
@@ -1,4 +1,8 @@
+
+#if USE_MYSQL
```

Comment:
Don't understand, why these ifdefs are here, in library, not in application code around usages of libmysqlxx library?